src/pages/services/aem-cf-console-admin/api/grid-columns/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Grid Columns - AEM Content Fragments Console Extensibility
-description: Learn what is possible to extend and customize in AEM Content Fragments console
+description: Learn what is possible to extend and customize in Grid Columns in AEM Content Fragments console
```

Comment:
Does not sound right on English. Maybe something like "Learn how to customize Grid Columns in AEM Content Fragments console"